manifest/cluster-network-15-nsx-secret.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-data: {tls.crt: null, tls.key: null}
+data: {tls.crt: "{{.NsxCert}}", tls.key: "{{.NsxKey}}", tls.ca: "{{.NsxCA}}"}
```

Comment:
Right now even operator puts CA cert in nsx-secret, but how does NCP read it? I think we lost the implementation on NCP side to read CA from secret, not sure how does QE test it before.

Reply:
Thanks, maybe later we can update this to README.